xml/System.Security.Claims/Claim.xml

```diff
@@ -87,9 +87,18 @@ if (null != principal)
         <Parameter Name="reader" Type="System.IO.BinaryReader" />
       </Parameters>
       <Docs>
-        <param name="reader">To be added.</param>
-        <summary>To be added.</summary>
-        <remarks>To be added.</remarks>
+        <param name="reader">A <see cref="T:System.IO.BinaryReader" /> pointing to a <see cref="T:System.Security.Claims.Claim" />.</param>
+        <summary>Initializes an instance of <see cref="T:System.Security.Claims.Claim" /> using a <see cref="T:System.IO.BinaryReader" />.</summary>
+        <remarks>
+          <format type="text/markdown"><![CDATA[  
+  
+## Remarks  
+ Normally, the reader is constructed from the bytes returned from <see cref="M:System.Security.Claims.Claim.WriteTo%2A" />. 
+  
+ ]]></format>
+        </remarks>
+        <exception cref="T:System.ArgumentNullException">
+          <paramref name="reader" /> is **null**.</exception>
```

Comment:
it totally sucks to do this without the tool. I think the null here needs to be like this since it's ECMAXML:
`<see langword="null" />`